### ISSUE TITLE
Css

### ISSUE THREAD

**Issue by EsturasLei** [gitdagray/css_course#7]

**Comment by CodPro-Sui** [comment]
No, you can write all capital letter { CSS-(cascading style sheet)}

**Comment by namanpunn** [comment]
<h1>Please go through this code if you are looking for the 2nd lecture css file</h1>html{
    background-color: black;
    font-size: 22px;
}
body{
    
}
h1,h2{
    color: blue;
}

.highlight{
    background-color: yellow;
    text-transform: uppercase;
}
.gray{
    color: gray;
}
.main{
    font-family: monospace;
    color:red;
}

#second{
    font-style:italic;
}

p {
    color: purple;
    
}
